bindings/scripts/compute_interfaces_info_individual.py

```diff
@@ -140,10 +140,21 @@ def get_put_forward_interfaces_from_definition(definition):
 
 
 def get_unforgeable_attributes_from_definition(definition):
+    # Legacy Python 2 way to sort lists. Group by type, and then sort by value.
+    class MultitypeSortKey:
```

Comment:
nit: Are there any tests that cover this expected legacy sorting mechanic? Up to you if you think it's worth adding.

Reply:
Hmm, I don't think so and I'm unsure what tests to write here as this is Chromium code. Without a sort key, the script fails altogether as Python 3 disallows sorts on multiple types, so it's necessary, but it's unclear what the previous expected sort behavior was.

There's some complication around completely emulating the Python 2 sorting behavior (https://stackoverflow.com/questions/26575183/how-can-i-get-2-x-like-sorting-behaviour-in-python-3-x for some discussion), but this gets close enough.

Reply:
Got it, thanks for the context!